examples/TestClient.java

```diff
@@ -0,0 +1,526 @@
+            System.out.println(x.getMessage());
+        }
+        // Order:: Comment(POST)
+        System.out.println("Order:: Comment(POST)");
+        try {
+            response = this.gc.postOrderComment(order_id, "Test order comment via API.");
+            System.out.print("[OK]");
+            System.out.println(verbose ? response.toString() : "");
+        } catch (GengoException e) {
+            System.out.print("[NG]");
+            System.out.println(x.getMessage());
+        }
+        // Order:: Comments(GET)
+        System.out.println("Order:: Comments(GET)");
+        try {
+            response = this.gc.getOrderComments(order_id);
+            System.out.print("[OK]");
+            System.out.println(verbose ? response.toString() : "");
+        } catch (GengoException e) {
+            System.out.print("[NG]");
+            System.out.println(x.getMessage());
+        }
+        // Order::(DELETE)
+        System.out.println("Order::(DELETE)");
+        try {
+            response = this.gc.deleteTranslationOrder(order_id);
+            System.out.print("[OK]");
+            System.out.println(verbose ? response.toString() : "");
+        } catch (GengoException e) {
+            System.out.print("[NG]");
+            System.out.println(x.getMessage());
+        }
+        System.out.println("[testOrder] end@" + this.sessionHash);
+    }
+
+    /**
+     * Test glossary functionalities.
+     */
+    public void testGlossary(Boolean verbose)
+    {
+        verbose = verbose == null ? true : verbose;
+        System.out.println("[testGlossary] start@" + this.sessionHash);
+        JSONObject response = null;
+        int glossary_id = -1;
+        // Glossaries (GET)
+        System.out.println("Glossaries::(GET)");
+        try {
+            response = this.gc.getGlossaryList();
+            glossary_id = response.getJSONArray("response").getJSONObject(0).getInt("id");
+            System.out.print("[OK]");
+            System.out.println(verbose ? response.toString() : "");
+        } catch (JSONException e) {
+            System.out.print("[NG]");
+            System.out.println(x.getMessage());
+        } catch (GengoException e) {
+            System.out.print("[NG]");
+            System.out.println(x.getMessage());
+        }
+        // Glossary (GET)
+        System.out.println("Glossary::(GET)");
+        if (glossary_id != -1) {
+            try {
+                response = this.gc.getGlossary(glossary_id);
+                System.out.print("[OK]");
+                System.out.println(verbose ? response.toString() : "");
+            } catch (GengoException e) {
+                System.out.print("[NG]");
+                System.out.println(x.getMessage());
+            }
+        } else {
+            System.out.println("[Skipped]");
+        }
+        System.out.println("[testGlossary] end@" + this.sessionHash);
+    }
+
+    public static void main(String[] args) throws GengoException {
```

Comment:
Because we don't make a deal about unit-test.
You can replace it to JUnit after this PR is merged if you want.